spark/src/main/scala/org/apache/spark/sql/delta/catalog/DeltaCatalog.scala

```diff
@@ -718,6 +720,18 @@ class DeltaCatalog extends DelegatingCatalogExtension
             val (oldField, pos) = getColumn(field)
             columnUpdates(field) = oldField.copy(name = rename.newName()) -> pos
 
+          case sync: SyncIdentity =>
+            syncIdentity = true
+            val field = sync.fieldNames
+            val (oldField, pos) = getColumn(field)
+            if (!ColumnWithDefaultExprUtils.isIdentityColumn(oldField)) {
+              throw DeltaErrors.identityColumnAlterNonIdentityColumnError()
+            }
+            // If the IDENTITY column does not allow explicit insert, high water mark should
```

Comment:
```suggestion
            // If the IDENTITY column does not allow explicit insert, high watermark should
```

Reply:
No, the way it was written before is correct. `water mark`, as two words. It's the way it is in the protocol for row tracking _and_ identity columns: https://github.com/delta-io/delta/blob/master/PROTOCOL.md#identity-columns

Reply:
True, you are right.